### PR TITLE
Fixes for French version

### PR DESCRIPTION
**Pull request by gpatrick2**
Multiple fixes for the French version, only errors left should be MD024 and MD036 as mentioned in the email earlier

Fix for MD001 is in the English version of section 1
Fixes for MD005 are in the English version of section 6

Double checked the errors for MD024 and MD036 in the English version, they should all be good

Added fixes to external links for the French version

### CODE REVIEW

**Comment by thomasgohard** [comment]
@gpatrick2 can you rebase your branch. Let me know if you need me to walk you through rebasing your branch.

**Comment by gpatrick2** [comment]
@thomasgohard should be good now. Let me know if anything else needs to be fixed

**Comment by thomasgohard** [comment]
@gpatrick2 can you resolve the merge conflicts.

Also, instead of creating a separate commit to remove the mdl report, it's cleaner if you do an interactive rebase of your branch to remove the commit that added it.

**Comment by gpatrick2** [comment]
@thomasgohard Paul had added a few changes after I rebased. There shouldn't be any merge conflicts now.

File: en/1-design-with-users.md [hunk 1]
```diff
@@ -40,8 +40,6 @@
 
 Users should be involved throughout the lifecycle of the service, with user research and testing informing the earliest design phases through to continuous improvements after the service has launched.
 
-**Think - Digital Design Playbook (ISED):** The absence of the client voice leads to assumptions that may be incorrect and costly. One way to challenge assumptions is to have quality qualitative and quantitative data. Data informed by client behaviours and needs will help you make the right decisions when designing and redesigning digital services. Service and digital performance and usage must be continuously measured in order to effectively direct transformation efforts. Client feedback should be collected and analyzed on an ongoing basis. Measurement and data analysis must remain integral to your service throughout its lifecycle.
```
Comment (pjackson28) [inline en/1-design-with-users.md]: This shouldn't be removed. It is new content that was added in a previous pull request.

File: en/1-design-with-users.md [hunk 2]
```diff
@@ -64,44 +62,8 @@ Users should be involved throughout the lifecycle of the service, with user rese
 
 - Use qualitative and quantitative data to help improve your understanding of user needs and identify areas for improvement **(OneGC product design criteria)**
 
-- Research what data is available **(Think - Digital Design Playbook (ISED))**
```
Comment (pjackson28) [inline en/1-design-with-users.md]: This stuff shouldn't be removed. It is new content that was added by a previous pull request.

**Comment by pjackson28** [comment]
This pull request seems to remove a lot of the new content that I added in a previous pull request. Please go over every change (can see under the Files tab) and undo any changes you didn't intend to do.

Should also squash the commits for adding and removing the CSV file.

**Comment by gpatrick2** [comment]
Fixes are in, I just have to get this conflict which I've fixed before to be accepted.

**Comment by pjackson28** [comment]
Most of the issues you were trying to fix should have been fixed by my last pull request (major cleanup of English and French content). I'm going to close this pull request since you'll need to start over. Please rerun the markdown quality test and create a new PR to address anything I missed (make sure to update your fork first though).

**Pull request closed without merging**